src/System.Reflection.Metadata/tests/PortableExecutable/PEBuilderTests.cs

```diff
@@ -95,21 +95,14 @@ private static unsafe void VerifyStrongNameSignatureDirectory(PEReader peReader,
             peBlob.WriteContentTo(peStream);
         }
 
-        public static IEnumerable<object[]> AllMachineTypes()
+        public static IEnumerable<object> AllMachineTypes()
         {
-            foreach (object enumValue in Enum.GetValues(typeof(Machine)))
+            return ((Machine[])Enum.GetValues(typeof(Machine))).Select(m => new object[]{(object)m});
+/*            foreach (object enumValue in Enum.GetValues(typeof(Machine)))
```

Comment:
Forgotten comments?

Reply:
Grumble... I had deleted those comments in my editor, but not actually saved them before committing. Sigh. Well, thanks for the signoff.